src/Tribe/REST/V1/Endpoints/Single_Event.php

```diff
@@ -560,12 +560,27 @@ protected function prepare_postarr( WP_REST_Request $request ) {
 			'EventCurrencySymbol'   => tribe( 'cost-utils' )->parse_currency_symbol( $request['cost'] ),
 			'EventURL'              => filter_var( $request['website'], FILTER_SANITIZE_URL ),
 			// Taxonomies
-			'tax_input'             => array_filter( array(
-				$events_cat => Tribe__Terms::translate_terms_to_ids( $request['categories'], $events_cat ),
-				'post_tag'  => Tribe__Terms::translate_terms_to_ids( $request['tags'], 'post_tag' ),
-			) ),
+			'tax_input'             => array(),
 		);
 
+		// Check if categories is provided (allowing for empty array to remove categories).
+		if ( isset( $request['categories'] ) ) {
+			$postarr[ $events_cat ] = array();
```

Comment:
I think ternary operator might be a good use case for this one to remove next `if` and `!` operation.

```php
$postarr[ $events_cat ] = empty( $request['categories'] ) 
	? array () 
	: Tribe__Terms::translate_terms_to_ids( $request['categories'], $events_cat );
```

Reply:
I'm not a fan of ternary operators with longer lines/function calls. In general, I try to avoid ternary operators entirely, it remains easier to read the code as it is, I think it'll be fine to not change it further.

Reply:
Yeah I think is something that does not have any difference other than personal preferences so I think this one should be 💯 🆗